.eslintrc.json

```diff
@@ -0,0 +1,268 @@
+{
+    "extends": "eslint:recommended",
```

Comment:
I'd rather not extend any built-in preset as they can change. We should define our own rules ourselves.